localtypings/pxtarget.d.ts

```diff
@@ -551,6 +551,7 @@ declare namespace pxt {
         usedBlocks: Map<number>;
         snippetBlocks: Map<Map<number>>;
         highlightBlocks: Map<Map<number>>;
```

Comment:
Not necessarily in this pr, but this could probably be merged in as just a field in `validateBlocks` (maybe would wanna rename it a bit more generic e.g. `taggedBlocks["highlight"]`) -- we'd have to do the `list -> map of counts` when accessing it but that is being done for `validateBlocks["exists"]` anyway / the calculation is quick enough, and we can dedup the logic a bit.